### PR TITLE
cockpit.js: Add debugging marking for translated strings

### PR DESCRIPTION
**Pull request by marusak**
I am still trying to understand how this would help, opening PR to demonstrate how we can do this.

Looks like this:
Screenshot from 2023-08-30 09-12-56

### CODE REVIEW

**Comment by tpardeshi** [comment]
Thanks @marusak for this pull request. 
Enabling pseudo-localization markers will make it easy in automation for referencing and locating strings in the application, when it is running in preferred target localized language. It will be used to test the marked strings are translated on UI or not. Sometimes there are few cases where the strings(ex. Name of product, Name of services, etc) displayed in English language, can be detected as a bug in automation but in reality those happen to be displayed in English only and not in localized language. So pseudo-localization markers makes it easy to deal with such test scenario in automation. It would be really helpful if these markers get implemented for Cockpit .Thanks !!

**Comment by marusak** [comment]
@martinpitt what do you think about this? Could we land this and just see what output the translation team will provide us with?

File: pkg/lib/cockpit.js [hunk 1]
```diff
@@ -3794,8 +3794,12 @@ function factory() {
         if (po_data) {
             const translated = po_data[key];
             if (translated?.[1])
-                return translated[1];
+                string = translated[1];
         }
+
+        if (window.localStorage.test_l10n === 'true')
```
Comment (martinpitt) [inline pkg/lib/cockpit.js:3800]: This is a hot codepath. Could this be read once and then cached? You'd then need a reload of the page after setting the storage flag, but I think that's acceptable.

**Review by martinpitt — CHANGES_REQUESTED** [review]
Thanks! I'm not super-convinced, but okay. This PR is already a month old, let's please refresh it and do the caching below.

**Comment by martinpitt** [comment]
@marusak is this still an issue?

**Review by martinpitt — APPROVED** [review]
This is indeed "meh", but let's see what they make from it. Thanks!

**Pull request merged**